packages/rolldown/package.json

```diff
@@ -114,9 +114,9 @@
     "fs-extra": "^11.2.0",
     "glob": "^10.4.1",
     "locate-character": "^3.0.0",
-    "lodash-es": "^4.17.21",
     "mri": "^1.2.0",
     "npm-run-all2": "^6.2.0",
+    "remeda": "^2.10.0",
```

Comment:
`remeda` is typed-friendly utils library like lodash. It provides strong-typed version of `object.keys`, `map`, `filter` etc..